crates/node-core/src/args/benchmark_args.rs

```diff
@@ -0,0 +1,62 @@
+//! clap [Args](clap::Args) for benchmark configuration
+
+use clap::Args;
+use std::path::PathBuf;
+
+/// Parameters for benchmark configuration
+#[derive(Debug, Args, PartialEq, Eq, Default, Clone)]
+#[command(next_help_heading = "Benchmark")]
+pub struct BenchmarkArgs {
+    /// Run the benchmark from a specific block.
+    #[arg(long, verbatim_doc_comment)]
+    pub from: Option<u64>,
+
+    /// Run the benchmark to a specific block.
+    #[arg(long, verbatim_doc_comment)]
+    pub to: Option<u64>,
+
+    /// Path to a JWT secret to use for the authenticated engine-API RPC server.
+    ///
+    /// This will perform JWT authentication for all requests to the other EL.
```

Comment:
other el?

Reply:
made it more clear, now it says "to the given engine rpc url"